src/refract/RenderJSONVisitor.cc

```diff
@@ -223,6 +225,61 @@ namespace refract
         assign(renderer.get());
     }
 
+    namespace {
```

Comment:
This functions are used just in `void RenderJSONVisitor::visit(const ArrayElement& e)`
So I mean it is logical to have then just before this method where are used